registry/base_registry.go

```diff
@@ -332,7 +384,7 @@ func (r *BaseRegistry) Subscribe(url *common.URL, notifyListener NotifyListener)
 			return
 		}
 
-		listener, err := r.facadeBasedRegistry.DoSubscribe(url)
+		listener, err := f(url)
```

Comment:
when do unSubsrcibe,this listener is closed,But this cycle can't exit.  Is this the right logic?

Reply:
i will check it soon

Reply:
check it again